README.md

```diff
@@ -172,7 +171,8 @@ npm run build_all
 ## Performance Tests
 
 Run `npm run build_perf` or `npm run perf` to run the performance tests with `protractor`.
-Run `npm run perf_micro` to run micro performance test benchmarking operator.
+
+Run `npm run perf_micro [operator]` to run micro performance test benchmarking operator.
```

Comment:
I think this was intended to be on a new line (just Markdown requires two new lines). I also added `[operator]` to make it obvious you can run performance tests on a single operator. I'm not sure the description here https://github.com/ReactiveX/rxjs/blob/master/CONTRIBUTING.md#micro is still accurate.

Also this section should probably mention that you need to run `npm run build_all` first.